libraryBuilds/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore all the wrappers below.
+gradle/
+gradlew.bat
+# Under this directory, we do in fact need to preserve local.properties!
```

Comment:
Comment should be clearer explanation:

    # local.properties is retained in this directory for XXX reason